website/docs/d/users.html.markdown

```diff
@@ -33,10 +33,29 @@ The following arguments are supported:
 
 -> **NOTE:** Either `user_principal_names`, `object_ids` or `mail_nicknames` should be specified. These _may_ be specified as an empty list, in which case no results will be returned.
 
+* `ignore_missing` - (Optional) Ignore missing users and return users that were found. The data source will still fail if no users are found. Defaults to false.
+
 ## Attributes Reference
 
 The following attributes are exported:
 
 * `object_ids` - The Object IDs of the Azure AD Users.
 * `user_principal_names` - The User Principal Names of the Azure AD Users.
 * `mail_nicknames` - The email aliases of the Azure AD Users.
+* `users` - An Array of Azure AD Users. Each `user` object consists of the fields documented below.
+
+___
+
+The `user` object consists of:
+
+* `id` - The Object ID of the Azure AD User.
+* `user_principal_name` - The User Principal Name of the Azure AD User.
+* `account_enabled` - `True` if the account is enabled; otherwise `False`.
+* `display_name` - The Display Name of the Azure AD User.
+* `mail` - The primary email address of the Azure AD User.
+* `mail_nickname` - The email alias of the Azure AD User.
+* `mail_nickname` - The email alias of the Azure AD User.
```

Comment:
It looks like `mail_nickname` was duplicated.

Reply:
Good spot, thanks!